src/main/kotlin/net/maizegenetics/phgv2/brapi/model/BasePagination.kt

```diff
@@ -0,0 +1,56 @@
+package model
+
+import kotlinx.serialization.Serializable
+import java.util.*
+
+/**
+ *
+ * @param pageSize The number of data elements returned, aka the size of the current page. If the requested page does not have enough elements to fill a page at the requested page size, this field should indicate the actual number of elements returned.
+ * @param totalCount The total number of elements that are available on the server and match the requested query parameters.
+ * @param totalPages The total number of pages of elements available on the server. This should be calculated with the following formula.   totalPages = CEILING( totalCount / requested_page_size)
+ */
+@Serializable
+open class BasePagination(
+        /* The number of data elements returned, aka the size of the current page. If the requested page does not have enough elements to fill a page at the requested page size, this field should indicate the actual number of elements returned. */
+        var pageSize: kotlin.Int = 20 // LCJ - I arbitrarily chose 20
```

Comment:
Can we remove the kotlin. bits?  This looks like a weird Java -> kotlin conversion.

Reply:
done